src/query/expression/src/values.rs

```diff
@@ -1156,18 +1156,18 @@ impl Column {
                 })
             }
             Column::Array(col) => {
-                if col.len() == 0 || col.values.len() == 0 {
+                if col.len() == 0 {
```

Comment:
No need to check underlying_column().values.len()?

Reply:
`ArrayColumn` only modifies the `offsets` in the slice operation, the `underlying_column` is unchanged, so checking the length of the `underlying_column` may give a wrong result.